index.html

```diff
@@ -208,6 +225,29 @@ <h2> Deliverables </h2>
         <div id="normative">
           <h3> Normative Specifications </h3>
           <dl class='todo'>
+            <dt class="spec"><a href="https://www.w3.org/TR/ttml3/">Timed Text
+                Markup Language 3 (TTML3)<br>
+              </a></dt>
+            <dd>This collection of specifications defines a content type that
```

Comment:
Change to "specification". All of the content of this item refers to an Editor's Draft (in the singular) which is not a collection. TTML3 is a single REC track document (=specification) deliverable. If you want, you can a new item after  Timed Text Markup Language 3 (TTML3) that has a title "Timed Text Markup Language 3 (TTML3) Modules", and then list the ones we expect to publish along with the TTML3 specification.

Reply:
Done.